plugin.py

```diff
@@ -128,40 +128,49 @@ def is_valid_for_view(self, view: sublime.View) -> bool:
     def _handle_log_message_notification(self, payload: CopilotPayloadLogMessage) -> None:
         pass
 
+    @notification_handler(NTFY_PANEL_SOLUTION)
+    def _handle_panel_solution_notification(self, payload: CopilotPayloadPanelSolution) -> None:
+        view = ViewPanelCompletionManager.find_view_by_panel_id(payload["panelId"])
+        if not view:
+            return
+
+        preprocess_panel_completions(view, [payload])
+
+        completion_manager = ViewPanelCompletionManager(view)
+        completions = completion_manager.completions
+        completions.append(payload)
+        completion_manager.completions = completions
```

Comment:
Maybe we could introduce `completion_manager.append_completion(payload)`?

Reply:
sounds good.

Reply:
Done in 0550292